ui/src/pages/sessions/session-columns.tsx

```diff
@@ -15,137 +15,140 @@ import { STRING, translate } from 'utils/language'
 export const columns: (projectId: string) => TableColumn<Session>[] = (
   projectId: string
 ) => [
-    {
```

Comment:
Changes here are mostly because this file was missing some formatting (specified by Prettier).